### PR TITLE
Jdk89acesses to common

### PR DESCRIPTION
**Pull request by AndreasEhret**

### CODE REVIEW

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java [hunk 1]
```diff
@@ -30,32 +30,33 @@
...
+    private final static Logger LOG = LoggerFactory.getLogger(SunMiscLauncher.class);
 
-    public static ImageIcon getSecureImageIcon(String resource) {
+    public static ImageIcon getSecureImageIcon(final String resource) {
         try {
-            Class clazz = Class.forName("sun.misc.Launcher");
-            Object obj  = clazz.newInstance();
-            Method m = clazz.getMethod("getClassLoader");
-            ClassLoader cl = (ClassLoader) m.invoke(obj);
+            final Class clazz = Class.forName("sun.misc.Launcher");
```
Comment (hendrikebbers) [inline common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java]: const string

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java [hunk 2]
```diff
@@ -30,32 +30,33 @@
...
+            final Object obj  = clazz.newInstance();
+            final Method m = clazz.getMethod("getClassLoader");
```
Comment (hendrikebbers) [inline common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java]: const string

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/JarIndexAccess.java [hunk 3]
```diff
@@ -31,48 +29,47 @@
...
         } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
             throw new RuntimeException(e);
         }
     }
 
-    public static JarIndexAccess getJarIndexImpl(JarFile jarFile) throws IOException, NoSuchMethodException, IllegalAccessException, InvocationTargetException {
-        Method method = jarIndexClass.getMethod("getJarIndex", JarFile.class);
-        Object o = method.invoke(null, jarFile);
+    public static JarIndexAccess getJarIndexImpl(final JarFile jarFile) throws IOException, NoSuchMethodException, IllegalAccessException, InvocationTargetException {
+        final Method method = jarIndexClass.getMethod("getJarIndex", JarFile.class);
```
Comment (hendrikebbers) [inline common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/JarIndexAccess.java]: const String

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java (hunk 1 above)
Comment (AndreasEhret) [reply common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java]: done

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java (hunk 2 above)
Comment (AndreasEhret) [reply common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/SunMiscLauncher.java]: done

File: common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/JarIndexAccess.java (hunk 3 above)
Comment (AndreasEhret) [reply common/src/main/java/net/adoptopenjdk/icedteaweb/jdk89access/JarIndexAccess.java]: done

**Pull request merged**